### PR TITLE
Adding contributing guide to the project

### PR DESCRIPTION
**Pull request by victor-heliomar**
In order to make easier to collaborate with the project I added a non-exhaustive contributing guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Welcome to the i18n Translation Generator contributing guide
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```
Comment (Israel-Laguan) [inline CONTRIBUTING.md:3]:
```suggestion
Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```

Not sure if this is accurate? Maybe this statement is valid in another repo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+To get an overview of the project, read the README file. Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests)
+
+## Getting started
+
+There are a lot of things that you could do to collaborate on open sources projects!. Will leave you some ideas about what could you do to help us to improve i18n translation generator
```
Comment (Israel-Laguan) [inline CONTRIBUTING.md:18]: NIT
```suggestion
There are a lot of things that you could do to collaborate on open sources projects!. Will leave you some ideas about what could you do to help us to improve i18n translation generator.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+If you spot any problem while using the project, see any error on the doc, or just wants to suggest changes to improve the project you can [create a new issue](https://github.com/victor-heliomar/i18n-translation-generator/issues/new/choose). Please make sure of check first that the issue doesn't exists already exists on the [issues list](https://github.com/victor-heliomar/i18n-translation-generator/issues)
+
+#### Solve an issue
+
+Scan through our existing issues to find one that interests you. If you find an issue to work on, you can leave a comment on the issue to leave us know that you're working on it and when you had developed your solution you can open a PR with a fix.
+
+#### Help to solve the community doubts
+
+If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/blob/dev/CODE_OF_CONDUCT.md)
```
Comment (Israel-Laguan) [inline CONTRIBUTING.md:32]: Maybe you can change the URL a little 
```suggestion
If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/CODE_OF_CONDUCT.md)
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### Making changes
+
+#### Cloning repository and writing code
+
+1. Fork the repository.
+2. Clone the repository forked.
+3. Install or update to Node.js, at the version specified in .npmrc file.
```
Comment (Israel-Laguan) [inline CONTRIBUTING.md:40]: NIT
```suggestion
3. Install or update to Node.js, at the version specified in `.npmrc` file.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+2. Improve the documentation
+  -  Documentation is a exhaustive work, create readable and clear documentations for all the project is not an easy task, and of course, will help to know the point of view of other people. That is why if you see that there is something that could be improved on the doc, you could suggest it as a PR or as an issue and it will help us a lot.
+3. Collaborate on side projects related to the project
+  -  We are currently working on some projects related to i18n translation generator in order to improve the way you create i18n translations the most possible. Projects in mind are:
+    1. Landing Page to explain easily the idea to developers.
+    2. Documentation Page in order to improve the doc of the project and make more easier the use of it.
+    3. Visual Studio Code Extension in order of create a user interface for users that doesn't feels comfortable with CLI Commands. Feel free of collaborate in any of them! Any help is appreciate!
```
Comment (Israel-Laguan) [inline CONTRIBUTING.md:74]: NIT:
Add the last sentence in another line
```suggestion
    3. Visual Studio Code Extension in order to create a user interface for users that doesn't feel comfortable with CLI Commands. 
    
Feel free to collaborate on any of them! We would appreciate any help!
```

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
I would suggest some text about "rules in this repo", for example:

- JS code style? What about using something like eslint/prettier and leaving the config files here?
- A list of DON'Ts, for example, "We don't allow political comments or nationalistic supremacy, racial or religious claims, or anything that can hurt people" or "Please keep PRs short, if you have more than 1000 LOC please consider separate in different PRs", etc.
- Some link about PR/code review/comments etiquette

File: CONTRIBUTING.md (hunk 1 above)
Comment (victor-heliomar) [reply CONTRIBUTING.md:3]: Thanks! My mistake

File: CONTRIBUTING.md (hunk 3 above)
Comment (victor-heliomar) [reply CONTRIBUTING.md:32]: Proposed link seems like is not valid
image

File: CONTRIBUTING.md (hunk 5 above)
Comment (victor-heliomar) [reply CONTRIBUTING.md]: Thanks! Done!

File: CONTRIBUTING.md (hunk 3 above)
Comment (victor-heliomar) [reply CONTRIBUTING.md]: But fixed the typo and added final end on each paragraph.- Thanks!

**Comment by victor-heliomar** [comment]
> I would suggest some text about "rules in this repo", for example:
> 
> * JS code style? What about using something like eslint/prettier and leaving the config files here?
> * A list of DON'Ts, for example, "We don't allow political comments or nationalistic supremacy, racial or religious claims, or anything that can hurt people" or "Please keep PRs short, if you have more than 1000 LOC please consider separate in different PRs", etc.
> * Some link about PR/code review/comments etiquette

Hi! thanks for help reviewing. I think config for eslint/prettier can be on another PR. I think it would be useful to leave config file's path here. Will take note about that to create the issue

About the DON'Ts list you mentioned something similar to code of conduct, I already added a file for that (I used a template of GitHub but sounds good to me) https://github.com/victor-heliomar/i18n-translation-generator/blob/dev/CODE_OF_CONDUCT.md

About "Some link about PR/code review/comments etiquette" sounds like it is more for the file of code of conduct, not sure of adding that here, in any case could add it on another PR

**Comment by victor-heliomar** [comment]
Added issues for https://github.com/victor-heliomar/i18n-translation-generator/issues/30 and https://github.com/victor-heliomar/i18n-translation-generator/issues/29 and added the note that the config file path should be put in this file

**Pull request merged**